bedrock/firefox/firefox_details.py

```diff
@@ -12,6 +13,11 @@
 class _ProductDetails(ProductDetails):
     bouncer_url = 'https://download.mozilla.org/'
 
+    # Note allizom.org is the production endpoint for SHA-1.
+    # It uses this because that's the only SHA-1 certificate
+    # we have that's usable. (SHA-1 certs can no longer be issued).
+    sha1_bouncer_url = 'https://download.mozilla.org/' if switch('disable-sha1-downloads') else 'https://download-sha1.allizom.org/'
```

Comment:
That's the thing though, you'd have to actively set the switch to have the tests fail. The default switch value should pass (like it's doing in circle). I agree with you that if someone did set the variable it would potentially be confusing as to why the test was failing though. Fixing it right is preferable.